examples/hellohttp/e2e-test.sh

```diff
@@ -42,20 +42,43 @@ function EXPECT_CONTAINS() {
 }
 
 if [[ -z "${1:-}" ]]; then
-  echo "Usage: $(basename $0) <kubernetes namespace> <language ...>"
+  echo "ERROR: None of execution type, kubernetes namspace and languages is provided!"
+  echo "Usage: $(basename $0) <'remote' or 'local' run> <kubernetes namespace> <language ...>"
+  exit 1
+fi
+
+local=false
+if [[ "$1" == "local" ]]; then
+  local=true
+elif [[ "$1" != "remote" ]]; then
+  echo "ERROR: Execution type must be either 'remote' or 'local'!"
+  echo "Usage: $(basename $0) <'remote' or 'local' run> <kubernetes namespace> <language ...>"
+  exit 1
+fi
+shift
+
+if [[ -z "${1:-}" ]]; then
+  echo "ERROR: None of kubernetes namspace and languages is provided!"
+  echo "Usage: $(basename $0) <'remote' or 'local' run> <kubernetes namespace> <language ...>"
   exit 1
 fi
 namespace="$1"
 shift
 if [[ -z "${1:-}" ]]; then
   echo "ERROR: Languages not provided!"
-  echo "Usage: $(basename $0) <kubernetes namespace> <language ...>"
+  echo "Usage: $(basename $0) <'remote' or 'local' run> <kubernetes namespace> <language ...>"
   exit 1
 fi
 
 get_lb_ip() {
-    kubectl --namespace="${namespace}" get service hello-http-staging \
-      -o jsonpath='{.status.loadBalancer.ingress[0].ip}'
+  # Determine the location of variable to retrieve the service IP address
```

Comment:
same, lets refactor this to util.sh if its the same for all e2e-test.sh files

Reply:
Done